lib/pengine/tags.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2020 the Pacemaker project contributors
+ *
+ * The version control history for this file may have further details.
+ *
+ * This source code is licensed under the GNU Lesser General Public License
+ * version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+ */
+
+#include <glib.h>
+#include <stdbool.h>
+
+#include <crm/common/util.h>
+#include <crm/pengine/internal.h>
+#include <crm/pengine/pe_types.h>
+
+gchar **
+pe__unames_with_tag(pe_working_set_t *data_set, const char *tag_name) {
```

Comment:
Tiny nitpick, the function definition style is to put the opening brace on its own line

Reply:
I'm sure I have done that wrong in plenty of other places.